### DISCUSSION TITLE
is this possible to be exploited for shimless-rma

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1268] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Enable APRO update from RootFS in Shimless RMA
Updates APRO from RootFS first in the Shimless RMA flow and fallbacks to USB drive when it fails. – ChromeOS

[#shimless-rma-apro-update-rootfs](chrome://flags/#shimless-rma-apro-update-rootfs)

**Comment by akabutnicer** [comment, marked as answer]
to be blatant, no. i doubt this can be useful
